actors/evm/src/interpreter/bytecode.rs

```diff
@@ -1,39 +1,22 @@
 #![allow(dead_code)]
 
-use {super::opcode::OpCode, crate::interpreter::output::StatusCode, std::ops::Deref};
+use {super::opcode::OpCode, std::ops::Deref};
 
 pub struct Bytecode<'c> {
     code: &'c [u8],
-    jumpdest: Vec<bool>,
 }
 
 impl<'c> Bytecode<'c> {
-    pub fn new(bytecode: &'c [u8]) -> Result<Self, StatusCode> {
-        // only jumps to those addresses are valid. This is a security
-        // feature by EVM to disallow jumps to arbitary code addresses.
-        // todo: create the jumpdest table only once during initial contract deployment
-        let mut jumpdest = vec![false; bytecode.len()];
-        let mut i = 0;
-        while i < bytecode.len() {
-            if bytecode[i] == OpCode::JUMPDEST as u8 {
-                jumpdest[i] = true;
-                i += 1;
-            } else if bytecode[i] >= OpCode::PUSH1 as u8 && bytecode[i] <= OpCode::PUSH32 as u8 {
-                i += (bytecode[i] - OpCode::PUSH1 as u8) as usize + 2;
-            } else {
-                i += 1;
-            }
-        }
```

Comment:
I believe this is required by EVM spec.

Reply:
we are still checking for jmpdests when jumping, we just dont do this extraneous thing that doesnt buy us anything.